[New Feature] Emphasized Inflows

42bottles opened the pull request:
New feature to highlight inflows green.

GitHub Issue (if applicable): #1780 

Trello Link (if applicable): https://trello.com/c/PGKbgaxc/533-option-for-all-inflows-to-be-green

**Explanation of Bugfix/Feature/Modification:**

New feature that changes colour of inflows on accounts page to green. Based off of existing Emphasize Outflows feature.

joshmadewell commented:
@42bottles sorry for the delay on this. I'm cool with adding this feature but can you add support for all the different themes by using css variables. You can also just change your own class definition based on the selected style (search for `theme-dark` to see a few examples of this)

42bottles commented:
Changed it to use the themes. At the moment they all have the same base colour, I think it looks fine across all themes. But I've changed the selected colour new-default to contrast better with the lighter selected row background.

_Sorry about the weird merging, still learning github and was just trying to get the files on my PC updated to the latest toolkit version._

joshmadewell commented:
Nice thanks for the new feature and it's no problem since I just squash it all into one commit anyway :D
